src/panoramic/cli/pano_model.py

```diff
@@ -56,6 +56,96 @@ def __eq__(self, o: object) -> bool:
         return self.to_dict() == o.to_dict()
 
 
+class PanoField(Actionable):
+    """Field definition."""
+
+    API_VERSION = 'v1'
+
+    def __init__(
+        self,
+        *,
+        slug: str,
+        group: str,
+        display_name: str,
+        data_type: str,
+        field_type: str,
+        description: Optional[str],
+        data_source: Optional[str],
+        calculation: Optional[str],
+        display_format: Optional[str],
+        aggregation: Optional[Dict[str, Any]],
```

Comment:
We should have a class defined for this, just like we do with joins etc. in `PanoModel`

Reply:
Done in 2e0af3a9380aa2ee124e37a8319df8f13e945f77.